Source/Modules/python.cxx

```diff
@@ -2376,21 +2376,22 @@ class PYTHON:public Language {
   void emitFunctionShadowHelper(Node *n, File *f_dest, String *name, int kw) {
     String *parms = make_pyParmList(n, false, false, kw);
     String *callParms = make_pyParmList(n, false, true, kw);
-    /* Make a wrapper function to insert the code into */
-    Printv(f_dest, "\ndef ", name, "(", parms, ")", returnTypeAnnotation(n), ":\n", NIL);
-    if (have_docstring(n))
-      Printv(f_dest, tab4, docstring(n, AUTODOC_FUNC, tab4), "\n", NIL);
-    if (have_pythonprepend(n))
-      Printv(f_dest, indent_pythoncode(pythonprepend(n), tab4, Getfile(n), Getline(n), "%pythonprepend or %feature(\"pythonprepend\")"), "\n", NIL);
-    if (have_pythonappend(n)) {
-      Printv(f_dest, tab4 "val = ", funcCall(name, callParms), "\n", NIL);
-      Printv(f_dest, indent_pythoncode(pythonappend(n), tab4, Getfile(n), Getline(n), "%pythonappend or %feature(\"pythonappend\")"), "\n", NIL);
-      Printv(f_dest, tab4 "return val\n", NIL);
+    String *retAnnotation = returnTypeAnnotation(n);
```

Comment:
You're right. This is something that I've accidentally left-over from my previous versions of the patch.